src/UnstableApis.ts

```diff
@@ -83,6 +93,139 @@ export class UnstableApis {
+
+    /**
+     * Gets the invited users of a group.
+     * @param {string} groupId The group ID of which to get the invited users.
+     * @return {Promise<any[]>} Resolves to an array of all the users invited to the group.
+     */
+    public async getGroupInvitedUsers(groupId: string): Promise<any[]> {
+        const response = await this.client.doRequest("GET", `/_matrix/client/r0/groups/${encodeURIComponent(groupId)}/invited_users`);
+        return response["chunk"];
+    }
+
+    /**
+     * Gets the rooms of a group.
+     * @param {string} groupId The group ID of which to get all the rooms.
+     * @return {Promise<any[]>} Resolves to an array of all the rooms of the group.
+     */
+    public async getGroupRooms(groupId: string): Promise<any[]> {
+        const response = await this.client.doRequest("GET", `/_matrix/client/r0/groups/${encodeURIComponent(groupId)}/rooms`);
+        return response["chunk"];
+    }
+
+    /**
+     * Accepts an invite to a group.
+     * @param {string} groupId The group ID of which to accept the invite of.
+     * @return {Promise<any>} Resolves when completed.
+     */
+    public async acceptGroupInvite(groupId: string): Promise<any> {
+        return this.client.doRequest("PUT", `/_matrix/client/r0/groups/${encodeURIComponent(groupId)}/self/accept_invite`, null, {});
+    }
+
+    /**
+     * Joins a group.
+     * @param {string} groupId The group ID to join.
+     * @return {Promise<any>} Resolves when completed.
+     */
+    public async joinGroup(groupId: string): Promise<any> {
+        return this.client.doRequest("PUT", `/_matrix/client/r0/groups/${encodeURIComponent(groupId)}/self/join`, null, {});
+    }
+
+    /**
+     * Leaves a group.
+     * @param {string} groupId The group ID of the group to leave.
+     * @return {Promise<any>} Resolves when completed.
+     */
+    public async leaveGroup(groupId: string): Promise<any> {
+        return this.client.doRequest("PUT", `/_matrix/client/r0/groups/${encodeURIComponent(groupId)}/self/leave`, null, {});
+    }
+
+    /**
+     * Sets the publicity of a group.
+     * @param {string} groupId The group ID to set the publicity of.
+     * @param {boolean} publicise If the group should be publicised.
+     * @return {Promise<any>} Resolves when completed.
+     */
+    public async setGroupPublicity(groupId: string, publicise: boolean): Promise<any> {
+        return this.client.doRequest("PUT", `/_matrix/client/r0/groups/${encodeURIComponent(groupId)}/self/update_publicity`, null, {
+            publicise,
+        });
+    }
+
+    /**
+     * Gets all group IDs joined.
+     * @return {Promise<string[]>} Resolves to the group IDs of the joined groups.
+     */
+    public async getJoinedGroups(): Promise<string[]> {
+        const response = await this.client.doRequest("GET", "/_matrix/client/r0/joined_groups");
+        return response["groups"];
+    }
+
+    /**
+     * Gets the group IDs that the spdecified user has publicised.
```

Comment:
```suggestion
     * Gets the group IDs that the specified user has publicised.
```

Reply:
I apparently don't have permission to apply this change.

Reply:
Fixed it!